ramls/check-in-by-barcode-response.json

```diff
@@ -67,6 +67,14 @@
             "name": {
               "description": "The name of the location",
               "type": "string"
+            },
+            "libraryName": {
```

Comment:
Rather than introduce library name as a property, could the name be nested in an object, in order to aid expansion to other library properties without cluttering the location object further